cmd/peer-rest-server.go

```diff
@@ -424,6 +424,20 @@ func (s *peerRESTServer) CPULoadInfoHandler(w http.ResponseWriter, r *http.Reque
 	logger.LogIf(ctx, gob.NewEncoder(w).Encode(info))
 }
 
+// CPUHWInfoHandler - returns CPU Hardware info.
+func (s *peerRESTServer) CPUHWInfoHandler(w http.ResponseWriter, r *http.Request) {
+	if !s.IsValid(w, r) {
```

Comment:
We should avoid inconsistent naming conventions here one is cpuhwinfo and another is HardwareCpuInfo . We should in my opinion just keep CpuInfo..

Reply:
done